.specify/memory/constitution.md

```
@@ -0,0 +1,42 @@
+- .specify/templates/plan-template.md ✅ updated
+- .specify/templates/spec-template.md ✅ updated
+- .specify/templates/tasks-template.md ✅ updated
+- .specify/templates/agent-file-template.md ✅ updated
+Follow-up TODOs: None
+-->
+
+# jsavrs Constitution
+
+## Core Principles
+
+### Safety First
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)